src/app/server/CommissioningWindowManager.cpp

```diff
@@ -334,8 +335,6 @@ CHIP_ERROR CommissioningWindowManager::StopAdvertisement(bool aShuttingDown)
     }
 #endif
 
-    mWindowStatus = AdministratorCommissioning::CommissioningWindowStatus::kWindowNotOpen;
```

Comment:
Once this is gone, I don't see a place where this state gets reset now. It seems like a good candidate would be the ResetState function, which gets called when CloseCommissioningWindow is called.

Reply:
The state get reset in ResetState now.

In case the commissioning complete.
OnPlatformEvent() -> Cleanup() ->  ResetState()

In case the commissioning timeout
CloseCommissioningWindow() -> Cleanup() ->  ResetState()

Reply:
I'm sorry - I honestly don't know how I missed that change.